core/client/client.go

```diff
@@ -74,6 +69,19 @@ var (
 	Vlog = log.New(ioutil.Discard, "", 0)
 )
 
+// Verifier is used to verify specific outputs from Key Transparency.
+type Verifier interface {
+	// Index computes the index from a VRF proof.
```

Comment:
What is VRF?

Reply:
VRF means [Verifiable Random Function](https://en.wikipedia.org/wiki/Verifiable_random_function) 

KT has a VRF implementation in `core/crypto/vrf/`